cloudigrade/config/settings/base.py

```diff
@@ -29,6 +29,8 @@
 # AWS Defaults
 S3_DEFAULT_REGION = env('S3_DEFAULT_REGION', default='us-east-1')
 SQS_DEFAULT_REGION = env('SQS_DEFAULT_REGION', default='us-east-1')
+HOUNDIGRADE_AVAILABILITY_ZONE = env('HOUNDIGRADE_AVAILABILITY_ZONE',
```

Comment:
Should we specifically call this an AWS AZ? Something like `HOUNDIGRADE_AWS_AVAILABILITY_ZONE`. Only reason I bring that up is in the spirit of being cloud agnostic, at some point we will have little houndigrades running around in azure and gcp and this could be confusing.